### PR TITLE
6.2.1 release notes

### PR DESCRIPTION
**Pull request by jbondpdx**

### CODE REVIEW

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -10,11 +10,21 @@ canonical: "/puppetserver/latest/release_notes.html"
...
 
+## Puppet Server 6.2.1
+
+Released 20 February 2019.
+
+This release contains resolved issues.
+
+### Resolved issues
+
+- This release upgrades Bouncy Castle to version 1.6.0 for security updates.
```
Comment (Magisus) [inline documentation/release_notes.markdown]: I think it's 1.60

Comment (jbondpdx) [reply documentation/release_notes.markdown]: This is fixed now

**Comment by justinstoller** [comment]
Hey, Jean can you preface your commit title with a `(docs)` or `(maint)` label. Like `(docs) 6.2.1 release notes`. Alternatively, we could do that in our "squash and merge" merge option if you've already moved on.

**Comment by jbondpdx** [comment]
Wow, I hope you weren't watching that mess as it happened, but I have learned some valuable lessons today. 🤔  Anyway, it's merged and the commit message is fixed!

**Pull request merged**